### PR TITLE
Add 1.7.2 to the CHANGELOG.md

### PR DESCRIPTION
**Pull request by Evi1Pumpkin**
## Target Release

1.7.2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## 1.7.2
+
+NEW FEATURES:
+
+ENHANCEMENTS:
+
+BUG FIXES:
+* Fix e2e test_test on darwin amd64 ([#1551](https://github.com/opentofu/opentofu/pull/1551))
```
Comment (ollevche) [inline CHANGELOG.md:8]: Do we think this type of change should be included in the changelog?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+* ensure authorization header is included for http backends ([#1656](https://github.com/opentofu/opentofu/pull/1656))
+* Fix bug in import 'to' parsing in json configurations ([#1665](https://github.com/opentofu/opentofu/pull/1665))
```
Comment (ollevche) [inline CHANGELOG.md:10]:
```suggestion
* Fixed: authorization header is not included for HTTP backends ([#1656](https://github.com/opentofu/opentofu/pull/1656))
* Fixed: bug in import 'to' parsing in JSON configurations ([#1665](https://github.com/opentofu/opentofu/pull/1665))
```

File: CHANGELOG.md (hunk 1 above)
Comment (cam72cam) [reply CHANGELOG.md:8]: We don't usually

Comment (Evi1Pumpkin) [reply CHANGELOG.md]: removed it

File: CHANGELOG.md (hunk 2 above)
Comment (Evi1Pumpkin) [reply CHANGELOG.md]: Done 👍

**Pull request merged**